Drink_Book_PR/DrinkBook/Club.cs

```diff
@@ -0,0 +1,186 @@
+            get
+            {
+                return name;
+            }
+            set
+            {
+                name = value;
+            }
+        }
+
+        public int PriceOfWhiskey
+        {
+            get
+            {
+                return priceOfWhiskey;
+            }
+            set
+            {
+                priceOfWhiskey = value;
+            }
+        }
+
+        public int PriceOfVodka
+        {
+            get
+            {
+                return priceOfVodka;
+            }
+            set
+            {
+                priceOfVodka = value;
+            }
+        }
+
+        public int Capacity
+        {
+            get
+            {
+                return capacity;
+            }
+            set
+            {
+                capacity = value;
+            }
+        }
+
+        public Club()
+        {
+            this.name = "";
+            this.priceOfWhiskey = 0;
+            this.priceOfVodka = 0;
+            this.capacity = 0;
+
+        }
+
+        public Club(string name, int priceOfWhiskey, int priceOfVodka, int capacity)
+        {
+            this.name = name;
+            this.priceOfWhiskey = priceOfWhiskey;
+            this.priceOfVodka = priceOfVodka;
+            this.capacity = capacity;
+        }
+
+        public virtual bool CanUserEnterTheClub(User user)
+        {
+            int priceOfAlcohol = ConvertDrinkableAlcoholToMoney(user.DrinkableWhiskeys, user.DrinkableVodkas);
+
+
+            if (user.Budget < priceOfAlcohol)
+            {
+                return false;
+            }
+            else if (this.listOfUsers.Count >= this.capacity)
+            {
+                return false;
+            }
+            else if (!(user.PreferredMusic == Constants.MUSIC_EVERYTHING || this.musicPlayed == user.PreferredMusic))
+            {
+                return false;
+            }
+
+            return true;
+        }
+
+        public int ConvertDrinkableAlcoholToMoney(int drinkableWhiskeys, int drinkableVodkas)
+        {
+            int fullPriceForAlcohol = 0;
+            int currentPriceOfWhiskey = 0;
+            int currentPriceOfVodka = 0;
+
+            currentPriceOfWhiskey = this.PriceOfWhiskey * drinkableWhiskeys;
+            currentPriceOfVodka = this.PriceOfVodka * drinkableVodkas;
+
+            fullPriceForAlcohol = currentPriceOfVodka + currentPriceOfWhiskey;
+
+            return fullPriceForAlcohol;
+        }
+
+        public List<User> ListOfUsers
+        {
+            get
+            {
+                return listOfUsers;
+            }
+            set
+            {
+                listOfUsers = value;
+            }
+        }
+
+        public List<User> AddUserToClub(User user)
+        {
+            if (CanUserEnterTheClub(user) == true)
+            {
+                listOfUsers.Add(user);
+            }
+            return listOfUsers;
+        }
+
+        public List<User> RemoveUserFromClub(User user)
```

Comment:
RemoveUser is enough, because Methods should not contain their class name as it is implied by it.